report/docker_run.py

```diff
@@ -363,15 +363,15 @@ def run_standard(cmd=None):
   experiment_name = f"{date}-{args.frequency_label}-{args.benchmark_set}"
 
   # Report directory uses the same name as experiment.
-  # See upload_report.sh on how this is used.
+  # See upload_report.py on how this is used.
   gcs_report_dir = f"{args.sub_dir}/{experiment_name}"
 
   # Trends report use a similarly named path.
   gcs_trend_report_path = f"{args.sub_dir}/{experiment_name}.json"
 
   # Generate a report and upload it to GCS
   report_process = subprocess.Popen([
-      "bash", "report/upload_report.sh", local_results_dir, gcs_report_dir,
+      "python3", "report/upload_report.py", local_results_dir, gcs_report_dir,
```

Comment:
Sorry that I missed this last time:
Could you please change this `python3` to be `python_path`  just like in
https://github.com/google/oss-fuzz-gen/blob/9771601a0e53c3281e6d4ade6f84f316ee208e1f/report/docker_run.py#L380

This is to ensure the correct `python` is used in the docker container.